api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,126 @@
+        "SMTP:outlookgroup101@service.microsoft.com"
+    ],
+    "securityEnabled": false,
+    "theme": null,
+    "visibility": "Public"
+}
+```
+Learn more about Office 365 groups and the administrator experiences [here](https://support.office.com/en-us/article/Learn-about-Office-365-groups-b565caa1-5c40-40ef-9915-60fdb2d97fa2).
+
+## Security groups
+Security groups are for controlling user access to resources. By checking whether or not a user is a member of a security group, your app can make authorization decisions when that user is trying to access some secure resources in your app. Security groups can have users and other security groups as members.
+
+### Example of security group
+
+```http
+{
+    "@odata.type": "#microsoft.graph.group",
+    "id": "f87faa71-57a8-4c14-91f0-517f54645106",
+    "deletedDateTime": null,
+    "classification": null,
+    "createdDateTime": "2016-07-20T09:21:23Z",
+    "description": "This group is a Security Group",
+    "displayName": "SecurityGroup101",
+    "groupTypes": [],
+    "mail": null,
+    "mailEnabled": false,
+    "mailNickname": "mysg",
+    "preferredLanguage": null,
+    "proxyAddresses": [],
+    "securityEnabled": true
+}
+```
+## Dynamic membership 
+All types of groups can have dynamic membership rules which automatically add or remove members from the group based on user properties. For example, a "Marketing employees" group would include every user with the department property set to "Marketing", so that new marketing employees are automatically added to the group and employees who leave the department are automatically removed from the group. This rule can be specified in a "membershipRule" field during group creation as ```"membershipRule": 'user.department -eq "Marketing"'``` GroupType must also include ```"DynamicMembership"```. The following request creates a new Office 365 group for the marketing employees: 
+
+```http
+POST https://graph.microsoft.com/v1.0/groups
+{
+    "description": "Marketing department folks",
+    "displayName": "Marketing department",
+    "groupTypes": [
+        "Unified",
+        "DynamicMembership"
+    ],
+    "mailEnabled": true,
+    "mailNickname": "marketing",
+    "securityEnabled": false,
+    "membershipRule": 'user.department -eq "Marketing"',
+    "membershipRuleProcessingState": "on"
+}
+```
+
+Learn more about formulating membershipRules in [advanced rules](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-groups-dynamic-membership-azure-portal).
+
+**NOTE** Dynamic membership rules requires the tenant to have a license at tier [Azure Active Directory Premium P1](https://azure.microsoft.com/en-us/pricing/details/active-directory/) or greater.
+
+## Common use cases
+
+Using Microsoft Graph, you can perform the these common operations and more:
+
+| **Use cases**		   | **REST resources**	| **See also** |
+|:---------------|:--------|:----------|
+| **Group object and methods** | | |

```

Comment:
I would actually like to see this table broken down further so that it could tell me what kinds of things are supported by specific kinds of groups: For example, if create a group had its own row, you could note that only Outlook and pure security groups can be created. If Add members had its own row, you could note that members can only be added for Outlook and security groups, If list members had its own row you could mention that this is supported across all groups, etc. (I'm not saying that every single thing needs its pwn row -- for example, almost all of the reads could probably be grouped together -- just that where there are diff levels of support for different types of groups, they should get their own row.

Also we should break out group functions (that are exposed on directoryObject) and mention the functions that return transitive membership (for security/dist groups anyway) vs. nav properties, like "members" that return direct membership -- note: this does not apply to Office 365 groups (AFAIK they only have direct members). It is important though for security groups and I get questions about it ...